docs/root/configuration/http_filters/tap_filter.rst

```diff
@@ -0,0 +1,122 @@
+.. _config_http_filters_tap:
+
+Tap
+===
+
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.tap.v2alpha.Tap>`
+* This filter should be configured with the name *envoy.filters.http.tap*.
+
+.. attention::
+
+  The tap filter is experimental and is currently under active development. There is currently a
+  very limited set of match conditions, output configuration, output sinks, etc. Capabilities will
+  be expanded over time and the configuration structures are likely to change.
+
+The HTTP tap filter is used to tap HTTP traffic. At a high level, the configuration is composed of
```

Comment:
Writing nit: This first sentence is technically correct, but it might benefit from a more explanatory style, e.g. "The HTTP tap filter is used to interpose on and record HTTP traffic" or something like that.